i6_models/parts/conformer/mhsa.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import Optional, Callable
+import torch
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass
+class ConformerMHSAV1Config(ModelConfiguration):
+    input_dim: int
+    """input dim and total dimension for query/key and value projections, should be dividable by `num_att_heads`"""
+    num_att_heads: int
+    """number of attention heads"""
+    att_weights_dropout: float = 0.1
```

Comment:
See discussion about defaults in other PRs: https://github.com/rwth-i6/i6_models/pull/4#discussion_r1189133116 and https://github.com/rwth-i6/i6_models/pull/6#discussion_r1196077265